try {
  console.log('success');
} catch (error) {
  console.log(error);
}

try {
  console.log('success');
} catch {
  console.log(error);
}
